pom.xml

```diff
@@ -827,6 +956,7 @@
           <compilerArgs>
             <arg>-Xpkginfo:always</arg>
           </compilerArgs>
+          <release>${java.version}</release>
```

Comment:
what is reason of such extra release versions ?

Reply:
yes sorry need to revert.

Comes from `org.openrewrite.java.migrate.UpgradeToJava17`.

This is missing considering the recipe. IDK why it is, and its not needed kind of, but im sure they have a good reason to add it.